### PR TITLE
Update software attestation records to reflect final name change

### PR DESCRIPTION
**Pull request by jmorrowomni**
- Replace SADR record with RSAA record
- Add CNAME record for development environment

#  #

## 🗣 Description ##

Add CNAME entry for the Software Attestation team to point to new Landing Zone 
ALB for both staging and development.  

## 💭 Motivation and context ##

The application name and acronym were never officially approved prior to this.  
This updates the record and the target to reflect the approved nomenclature:
Repository for Software Attestations and Artifacts (RSAA)

## 🧪 Testing ##

DNS resolution will be verified after the record is created.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: route53_rsaa_app.tf [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+resource "aws_route53_record" "rsaa_dev_CNAME" {
+  provider = aws.route53resourcechange
+  name     = "dev.rsaa.${aws_route53_zone.cyber_dhs_gov.name}"
+  records  = ["rsaa-dev-674180318.us-gov-west-1.elb.amazonaws.com"]
+  ttl      = 300
+  type     = "CNAME"
+  zone_id  = aws_route53_zone.cyber_dhs_gov.zone_id
+}
+# ------------------------------------------------------------------------------
```
Comment (dav3r) [inline route53_rsaa_app.tf]: Not a big deal, but we typically would put a blank line in here for separation.
```suggestion
}

# ------------------------------------------------------------------------------
```

**Review by dav3r — APPROVED** [review]
LGTM! 👍

**Comment by dav3r** [comment]
@jmorrowomni Let me know if you'd like me to apply this in Production to verify that it looks good.

**Review by jsf9k — APPROVED** [review]
I agree with @dav3r's one minor suggestion, but otherwise I heartily approve.

File: route53_rsaa_app.tf (hunk 1 above)
Comment (jmorrowomni) [reply route53_rsaa_app.tf]: Apologies, I thought I did!  Suggestion taken and applied

**Comment by jmorrowomni** [comment]
> @jmorrowomni Let me know if you'd like me to apply this in Production to verify that it looks good.

At your leisure, yes please!

**Comment by dav3r** [comment]
> > @jmorrowomni Let me know if you'd like me to apply this in Production to verify that it looks good.
> 
> At your leisure, yes please!

The changes have been applied - please verify that everything looks as expected.

**Comment by jmorrowomni** [comment]
image
image

Everything looks good!

**Pull request merged**